emulators/xhyve/Portfile

```diff
@@ -19,16 +19,13 @@ long_description \
     It can run FreeBSD and vanilla Linux distributions and \
     may gain support for other guest operating systems in the future.
 
-checksums           rmd160  b1e803cad9b1577925bee4230efee8a1a198cea0 \
-                    sha256  8bc4198bea593f56676b17a36a2a2b1b6490f7aa26c0e06d70c501b79db3d77d \
-                    size    11718059
+checksums           rmd160  a8de75267e1ff10da9d9957aa68f582867fae65b \
+                    sha256  403b43e4d157ccdd5275fcce034afeb0f8a7804c029d7dcf670f1dcc7929a55f \
+                    size    11718076
 
-post-patch {
-    foreach script [glob -- ${worksrcpath}/${name}run-*.sh] {
-        reinplace "s|test/|${prefix}/share/${name}/test/|" ${script}
-        reinplace "s|build/${name}|${prefix}/bin/${name}|" ${script}
-    }
-}
```

Comment:
why are these `reinplaces` not necessary anymore?

Reply:
@reneeotten when I tried to build it locally, I have a warning:
```
--->  Extracting xhyve
Warning: reinplace s|build/xhyve|/opt/local/bin/xhyve| didn't change anything in /opt/local/var/macports/build/_Users_catap_ports_emulators_xhyve/xhyve/work/xhyve-83516a009c692ea5d2993d1071e68d05d359b11e/xhyverun-tinycorelinux.sh
Warning: reinplace s|build/xhyve|/opt/local/bin/xhyve| didn't change anything in /opt/local/var/macports/build/_Users_catap_ports_emulators_xhyve/xhyve/work/xhyve-83516a009c692ea5d2993d1071e68d05d359b11e/xhyverun-freebsd.sh
Warning: reinplace s|test/|/opt/local/share/xhyve/test/| didn't change anything in /opt/local/var/macports/build/_Users_catap_ports_emulators_xhyve/xhyve/work/xhyve-83516a009c692ea5d2993d1071e68d05d359b11e/xhyverun-windows.sh
Warning: reinplace s|build/xhyve|/opt/local/bin/xhyve| didn't change anything in /opt/local/var/macports/build/_Users_catap_ports_emulators_xhyve/xhyve/work/xhyve-83516a009c692ea5d2993d1071e68d05d359b11e/xhyverun-windows.sh
--->  Configuring xhyve
```

as far as I see it do not needed since https://github.com/machyve/xhyve/commit/1dd9a5165848c7ed56dafc41932c553ea56a12af#diff-a282a5eb55d8da9fb9e95d97048f60dd145519306facc437d60813da23df091e

Reply:
yes, but there are three of `.sh` scripts and therefore a total of 6 reinplaces that are executed - apparently, four of these don't do anything, but two of them do. Of course you can figure out which reinplaces you need to keep.

Reply:
@reneeotten god catch! Reverting back.